### PR TITLE
chore(Alerts): Ordered alphabetically the list of integrations

### PR DESCRIPTION
**Pull request by nbaenam**
Ordered the list of integrations on the [Alerts and applied intelligence notification integrations](https://docs.newrelic.com/docs/alerts-applied-intelligence/notifications/notification-integrations/) page

### CODE REVIEW

File: src/content/docs/alerts-applied-intelligence/notifications/notification-integrations.mdx [hunk 1]
```diff
@@ -622,165 +400,392 @@ The **Preview** section on the right hand-side shows an expected payload after t
...
+
+        * **Destination Name:** Custom name to identify the destination.
 
-    2. Enter the following information:
+        * **Domain:** The URL of the destination.
 
-        * **Name**: Custom name to identify the destination.
+        * **User-name:** The name of the user.
 
-        * **API Key**: For this integration, you will be asked to provide a REST API Key. There're two types of REST API keys in PagerDuty, [general access](https://support.pagerduty.com/docs/generating-api-keys#section-generating-a-general-access-rest-api-key) and [user tokens](https://support.pagerduty.com/docs/generating-api-keys#section-generating-a-personal-rest-api-key).
```
Comment (x8a) [inline src/content/docs/alerts-applied-intelligence/notifications/notification-integrations.mdx:633]: Is this not needed anymore?

File: src/content/docs/alerts-applied-intelligence/notifications/notification-integrations.mdx [hunk 2]
```diff
@@ -622,165 +400,392 @@ The **Preview** section on the right hand-side shows an expected payload after t
...
 
 
-  ### Two-way sync [#two-way-pagerduty]
+  ### Two-way sync [#two-way-snow]
 
-  To enable two-way sync, turn on the **two-way integration** toggle.
+  To configure the two-way integration:
 
-  When turned on, a PagerDuty subscription is created at a later stage for the selected PagerDuty service (see [customize a message template](#message-pagerduty)). The webhook contains access details to New Relic (URL and New Relic API key).
+      1. Turn on the `two-way integration` toggle.
```
Comment (x8a) [inline src/content/docs/alerts-applied-intelligence/notifications/notification-integrations.mdx:569]: I'd bold the UI text: **two-way integration**

File: src/content/docs/alerts-applied-intelligence/notifications/notification-integrations.mdx [hunk 3]
```diff
@@ -622,165 +400,392 @@ The **Preview** section on the right hand-side shows an expected payload after t
...
 
-  By default, any change of state to the PagerDuty incident created by New Relic will sync back to New Relic.
+      2. Open and download this <a href="https://storage.googleapis.com/newrelic-notifications/snow-two-way-sync/Newrelic-Snow-Sync-Configurations.xml" download="NewRelic-SNow-Sync-Configurations.xml">XML file</a>, which includes the business rule triggering events back to New Relic.
```
Comment (x8a) [inline src/content/docs/alerts-applied-intelligence/notifications/notification-integrations.mdx:571]: Shouldn't we format this as a MD link, instead of an HTML one? `[text](link)`

File: src/content/docs/alerts-applied-intelligence/notifications/notification-integrations.mdx [hunk 4]
```diff
@@ -622,165 +400,392 @@ The **Preview** section on the right hand-side shows an expected payload after t
...
 
-  <Callout variant="important">
-    Please note that if you're a PagerDuty Event Intelligence or Digital Operations customer using Intelligent Alert Grouping on a given service, there could be a potential mismatch in PagerDuty Incidents sent back to New Relic.
-  </Callout>
+      3. In the ServiceNow sidebar menu, go to **System Definition > Business Rules**.
```
Comment (x8a) [inline src/content/docs/alerts-applied-intelligence/notifications/notification-integrations.mdx:573]: Shouldn't this be: `**System Definition** > **Business Rules**`?

File: src/content/docs/alerts-applied-intelligence/notifications/notification-integrations.mdx [hunk 5]
```diff
@@ -622,165 +400,392 @@ The **Preview** section on the right hand-side shows an expected payload after t
...
 
+      4. Click the menu icon in one of the column headers, select **Import XML**, and upload the XML file you downloaded.
 
-  ### Sync with New Relic workflows [#two-way-sync-workflows]
 
-  * The closure of a New Relic issue is triggered when the PagerDuty incident is resolved.
+  Once the destination is saved, a New Relic API key is kept in `api_key_credentials`. The key is sent in a header as part of the callback REST call to New Relic.
```
Comment (x8a) [inline src/content/docs/alerts-applied-intelligence/notifications/notification-integrations.mdx:578]: I'd say saved, instead of kept.
Also, in the second sentence, I'd say:
```
The key is sent in a header as part of the REST callback call to New Relic.
```

File: src/content/docs/alerts-applied-intelligence/notifications/notification-integrations.mdx [hunk 6]
```diff
@@ -622,165 +400,392 @@ The **Preview** section on the right hand-side shows an expected payload after t
...
 
-  * The acknowledgment of a New Relic issue is triggered when the PagerDuty incident is acknowledged.
+  
+  ### Sync with workflows [#two-way-sync-snow-workflows]
 
+      * Closure of New Relic issue is triggered when the ServiceNow incident state changes to resolved.
```
Comment (x8a) [inline src/content/docs/alerts-applied-intelligence/notifications/notification-integrations.mdx:583]: I'd rephrase this in active voice.

File: src/content/docs/alerts-applied-intelligence/notifications/notification-integrations.mdx [hunk 7]
```diff
@@ -622,165 +400,392 @@ The **Preview** section on the right hand-side shows an expected payload after t
...
+      * Acknowledgment of New Relic Issue is triggered when the Servicnow incident state changes from open.
```
Comment (x8a) [inline src/content/docs/alerts-applied-intelligence/notifications/notification-integrations.mdx:584]: Same as above, and there's a typo in ServiceNow.

File: src/content/docs/alerts-applied-intelligence/notifications/notification-integrations.mdx [hunk 8]
```diff
@@ -622,165 +400,392 @@ The **Preview** section on the right hand-side shows an expected payload after t
...
 
-  ### Configure the message template in Workflows [#message-pagerduty]
 
-  To configure the message template:
+  ### Configure the message template in workflows [#configure-servicenow-message-template]
 
-    1. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Alerts & AI > Workflows** and click on an existing workflow, or the **+ Add a new Workflow** button and choose the PagerDuty notifier.
+  Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Alerts & AI > Workflows** and click on an existing workflow, or the **+ Add a new workflow** button, and choose a ServiceNow destination.
```
Comment (x8a) [inline src/content/docs/alerts-applied-intelligence/notifications/notification-integrations.mdx:589]: Why did we remove the step number?

File: src/content/docs/alerts-applied-intelligence/notifications/notification-integrations.mdx [hunk 9]
```diff
@@ -622,165 +400,392 @@ The **Preview** section on the right hand-side shows an expected payload after t
...
     
-    2. Choose a destination. You can also create a new destination at this stage.
-
-    3. Select a PagerDuty service.
+  Upon a successful connection, ServiceNow incident table columns are fetched from your account and automatically mapped to your ServiceNow instance.
```
Comment (x8a) [inline src/content/docs/alerts-applied-intelligence/notifications/notification-integrations.mdx:591]: Same as above, why no step number?

File: src/content/docs/alerts-applied-intelligence/notifications/notification-integrations.mdx [hunk 10]
```diff
@@ -622,165 +400,392 @@ The **Preview** section on the right hand-side shows an expected payload after t
...
+  When you add a custom value for a supported field, you can add a dynamic value from the issue payload or write your own value.
 
+  You can remove any non-required field and add your own.
 
- 
-  ### Send a test notification [#test-notification-pagerduty]
+    ### Send a test notification [#send-servicenow-test]
 
-  You can see how the PagerDuty incident will appear by clicking a test notification with default field values. If successful, a incident will be created and a link will appear.
+    Click on **Send test notification** to see ServiceNow incidents with default field values. If successful, there is a link with the created incident.
```
Comment (x8a) [inline src/content/docs/alerts-applied-intelligence/notifications/notification-integrations.mdx:601]:
```
  If successful, you'll see a link with the created incident.
```

File: src/content/docs/alerts-applied-intelligence/notifications/notification-integrations.mdx [hunk 11]
```diff
@@ -622,165 +400,392 @@ The **Preview** section on the right hand-side shows an expected payload after t
...
-  This integration requires to set up a New Relic PagerDuty integration in the service you would like New Relic to create incidents in.
+  <Collapser
+    className="freq-link"
+    id="slack"
+    title="Slack"
+  >
+  Send notification messages to your Slack channels.
 
-  To create a New Relic integration in the PagerDuty service follow these steps:
+  See how [to migrate from your old Slack webhook destination to the new Slack app](#migrate-slack) if you need more information about it.
```
Comment (x8a) [inline src/content/docs/alerts-applied-intelligence/notifications/notification-integrations.mdx:623]:
```
  See how to [migrate from your old Slack webhook destination to the new Slack app](#migrate-slack), for more information.
```

File: src/content/docs/alerts-applied-intelligence/notifications/notification-integrations.mdx [hunk 12]
```diff
@@ -622,165 +400,392 @@ The **Preview** section on the right hand-side shows an expected payload after t
...
+  ### Prerequisites [#slack-prereqs]
 
-    3. Find and mark the New Relic integration in the list and click  **Add**.
+  Your Slack workspace needs to have the [New Relic application](https://newrelic.slack.com/apps/AP92KQJS3-new-relic?tab=more_info)(or the [EU App](https://slack.com/apps/AS5D75HQQ-new-relic-eu?tab=more_info) for one.eu.newrelic customers) installed. A workspace administrator must approve the application before you can individually install it. Please contact support@newrelic.com for app support.
 
-          <img title="PagerDuty New Relic integration." width="70%;" alt="New Relic integration." src={accountsPagerDutyAddNewRelicIntegration}/>
+  ### Set up a Slack destination [#set-slack-destination]
 
-    4. Click on the right to view and copy your  **Integration Key**.
+  Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Alerts & AI**, click **Destinations**, and then select **Slack**.
```
Comment (x8a) [inline src/content/docs/alerts-applied-intelligence/notifications/notification-integrations.mdx:636]: Why did we remove the step numbers?

Also, I wouldn't bold the `>` symbol.

File: src/content/docs/alerts-applied-intelligence/notifications/notification-integrations.mdx [hunk 13]
```diff
@@ -622,165 +400,392 @@ The **Preview** section on the right hand-side shows an expected payload after t
...
+      width="80%;"
+      title="authentication completed successfully."
+      alt="A screenshot completed authentication."
+      src={accountsSlackAuthenticationCompleted}
+    />
 
+  ### Configure the Slack message template in Workflows [#configure-slack-message-settings]
 
-  Note that custom details in PagerDuty Alerts is automatically populated.
+  Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Alerts & AI > Workflows** and click on an existing workflow, or the **+ Add a workflow** button.
```
Comment (x8a) [inline src/content/docs/alerts-applied-intelligence/notifications/notification-integrations.mdx:676]: Why don't we number the steps?

File: src/content/docs/alerts-applied-intelligence/notifications/notification-integrations.mdx [hunk 14]
```diff
@@ -622,165 +400,392 @@ The **Preview** section on the right hand-side shows an expected payload after t
...
-    id="mobile_push"
-    title="New Relic mobile push"
+    id="Splunk_on_call_templates"
+    title="Splunk on-call (VictorOps)"
   >
-   Send push notifications to your New Relic iOS or Android mobile apps.
-    
+    ### Webhook template for Splunk On-call
+
+     Send notifications from Workflows to Splunk On-call using a webhook template
```
Comment (x8a) [inline src/content/docs/alerts-applied-intelligence/notifications/notification-integrations.mdx:699]:
```
     Send notifications from Workflows to Splunk On-call using a [webhook template](https://help.victorops.com/knowledge-base/new-relic-integration-guide/#about-splunk-on-call-and-new-relic).
```

File: src/content/docs/alerts-applied-intelligence/notifications/notification-integrations.mdx [hunk 15]
```diff
@@ -622,165 +400,392 @@ The **Preview** section on the right hand-side shows an expected payload after t
...
 
-    ### Set up a mobile push destination [#set-mobile_push-destination]
+  ### Set up a webhook destination [#set-webhook-destination]
 
-    To create a mobile push destination, you need the following:
+  To create a webhook destination follow these steps:
 
-    * **Push destination name:** A unique destination name.
-    * **User id:** This will be automatically filled in based on the current logged in user.
+  1. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Alerts & AI**, click **Destinations**, and then select **Webhook**.
```
Comment (x8a) [inline src/content/docs/alerts-applied-intelligence/notifications/notification-integrations.mdx:719]: I wouldn't bold the `>` symbol.

File: src/content/docs/alerts-applied-intelligence/notifications/notification-integrations.mdx [hunk 16]
```diff
@@ -622,165 +400,392 @@ The **Preview** section on the right hand-side shows an expected payload after t
...
+
+      1. Set a name for the template.
+      2. Select a pre-configured destination from the destinations list or create a new one.
+      3. Add custom headers (optional).
+      4. Configure the request's payload.
+
+
+### Customize the webhook payload in Workflows [#customize-webhook-payload]
     
+Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Alerts & AI > Workflows** and click on an existing workflow, or the **+ Add a new workflow** button and select a Webhook destination.
```
Comment (x8a) [inline src/content/docs/alerts-applied-intelligence/notifications/notification-integrations.mdx:754]: I wouldn't bold the `>` symbol.

File: src/content/docs/alerts-applied-intelligence/notifications/notification-integrations.mdx [hunk 17]
```diff
@@ -622,165 +400,392 @@ The **Preview** section on the right hand-side shows an expected payload after t
...
+      The request's content-type is JSON by default, so the payload needs to be in JSON format too. To get familiar with the format, see [our usage examples](/docs/alerts-applied-intelligence/notifications/message-templates/#usage-examples). 
     </Callout>
     
+You can use the default payload or customize it to contain the required data. Pick variables from the [variables menu](/docs/alerts-applied-intelligence/notifications/message-templates/#variables-menu) and apply [handlebars syntax](/docs/alerts-applied-intelligence/notifications/message-templates/#handlebars-syntax) to enrich your webhook.
 
-  ### Configure when to receive push notifications in workflows [#mobile]
+The **Preview** section on the right hand-side shows an expected payload after the template is rendered. If the payload doesn't form a valid JSON, there'll be an error and you couldn't save the template.
```
Comment (x8a) [inline src/content/docs/alerts-applied-intelligence/notifications/notification-integrations.mdx:762]:
```suggestion
The **Preview** section on the right hand-side, shows an expected payload after the template is rendered. If the payload doesn't form a valid JSON, there'll be an error and you won't save the template.
```

File: src/content/docs/alerts-applied-intelligence/notifications/notification-integrations.mdx [hunk 18]
```diff
@@ -622,165 +400,392 @@ The **Preview** section on the right hand-side shows an expected payload after t
...
+       />
 
-  <Collapser
-    className="freq-link"
-    id="opsgeniewebhook_templates"
-    title="Opsgenie"
-  >
-    ### Webhook template for Atlassian Opsgenie
+  <Callout variant="tip">
+    Undefined type errors may indicate that the attribute wasn't indexed recently or doesn't exist. To fix the error, try adding an `if else` statement. For example, ` "closed_at": {{#if issueClosedAtUtc}} {{ json issueClosedAtUtc }} {{else}}"None"{{/if}} `
```
Comment (x8a) [inline src/content/docs/alerts-applied-intelligence/notifications/notification-integrations.mdx:772]:
```
Undefined type errors may indicate that the attribute wasn't indexed recently or doesn't exist. To fix the error, try adding an `if else` statement. For example:
`"closed_at": {{#if issueClosedAtUtc}} {{ json issueClosedAtUtc }} {{else}}"None"{{/if}}`

```

**Review by x8a — COMMENTED** [review]
I left some comments, but it all looks good to me 💃🏻.

File: src/content/docs/alerts-applied-intelligence/notifications/notification-integrations.mdx (hunk 8 above)
Comment (nbaenam) [reply src/content/docs/alerts-applied-intelligence/notifications/notification-integrations.mdx:589]: Added

File: src/content/docs/alerts-applied-intelligence/notifications/notification-integrations.mdx (hunk 9 above)
Comment (nbaenam) [reply src/content/docs/alerts-applied-intelligence/notifications/notification-integrations.mdx:591]: Added

File: src/content/docs/alerts-applied-intelligence/notifications/notification-integrations.mdx (hunk 13 above)
Comment (nbaenam) [reply src/content/docs/alerts-applied-intelligence/notifications/notification-integrations.mdx:676]: Added
